app/assets/stylesheets/collection.scss

```diff
@@ -1,3 +1,8 @@
 .collection-result {
   padding: 10px 5px;
+
+  .collection-info {
+    font-style: italic;
+    color: grey;
```

Comment:
Perhaps we could use a `<em class="text-muted">` instead of this custom styling.